src/pages/UserAdmin/index.tsx

```diff
@@ -75,6 +76,7 @@ const UserAdmin = () => {
     <Page banner={<PrimaryTopBox />} options={{ title: 'Brukeradmin' }}>
       <Paper className={classes.content}>
         <Typography variant='h1'>Brukeradmin</Typography>
+        <p>{members}</p>
```

Comment:
Bruk alltid `Typography` for tekst
```suggestion
        <Typography>{members}</Typography>
```